packages/flutter_tools/lib/src/commands/screenshot.dart

```diff
@@ -6,17 +6,37 @@ import 'dart:async';
 import 'dart:io';
 
 import 'package:flutter_tools/src/device.dart';
+import 'package:http/http.dart' as http;
 import 'package:path/path.dart' as path;
 
 import '../base/utils.dart';
 import '../globals.dart';
 import '../runner/flutter_command.dart';
 
+const String _kOut = 'out';
+const String _kSkia = 'skia';
+const String _kSkiaServe = 'skiaserve';
+
 class ScreenshotCommand extends FlutterCommand {
   ScreenshotCommand() {
-    argParser.addOption('out',
+    argParser.addOption(
+      _kOut,
       abbr: 'o',
-      help: 'Location to write the screenshot.');
+      help: 'Location to write the screenshot.',
+    );
+    argParser.addOption(
+      _kSkia,
+      valueHelp: 'port',
+      help: 'Retrieve the last frame rendered by a Flutter app as a Skia picture\n'
+        'using the specified diagnostic server port.\n'
+        'To find the diagnostic server port number, use "flutter run --verbose"\n'
+        'and look for "Diagnostic server listening on" in the output.'
```

Comment:
Agreed. Not great to direct users to the verbose logs. One way to address this is to make the default diagnostic port have a default value rather than auto-selecting what appears to be a random port. Then, most of the time(?), users would not need to specify a skia port.